docs/docs/guides/labs/components/building-pipelines-with-components/adding-attributes-to-assets.md

```diff
@@ -0,0 +1,34 @@
+---
+title: 'Adding attributes to assets in a subdirectory'
+sidebar_position: 550
+---
+
+import Preview from '@site/docs/partials/\_Preview.md';
+
+<Preview />
+
+It can be useful to apply configuration to assets within a specific component directory. Because each level in the hierarchy is implicitly a `dagster.components.DefsFolderComponent`, you can configure the entire subdirectory by adding a `component.yaml` file at the desired level in the hierarchy.
```

Comment:
I think this detail should be deferred to the bottom of the document. It's a confusing detail for someone just trying to copy paste something quickly or something.

Reply:
By this detail I mean the implicit `dagster.components.DefsFolderComponent` detail